src/managers/UserAccessManager/useUserAccessManagerActions.js

```diff
@@ -91,10 +89,24 @@ export function useUserAccessManagerActions() {
 				enable: user.disabled ? '1' : '',
 			},
 		});
-
-		openLegacyModal({title: t('grid.user.disable')}, (closeData) => {
-			finishedCallback();
+		let currentRoles = '';
+		user.groups.forEach((group) => {
+			currentRoles = currentRoles + group.name + ',';
 		});
+
+		openLegacyModal(
+			{
+				title: !user.disabled
+					? t('user.disabledModal.title', {fullName: user.fullName})
+					: t('user.enabledModal.title', {fullName: user.fullName}),
+				description: t('user.disabledModal.description', {
+					roles: currentRoles.replace(/,\s*$/, ''),
```

Comment:
Given suggestion above, we can remove the need to use regex and replacing string:
```suggestion
					roles: currentRoles,
```